dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/Admin/AddUserImport.cshtml.cs

```diff
@@ -0,0 +1,119 @@
+using System.ComponentModel.DataAnnotations;
+using System.Globalization;
+using CsvHelper;
+using CsvHelper.Configuration;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using TeacherIdentity.AuthServer.Infrastructure.Security;
+using TeacherIdentity.AuthServer.Models;
+using TeacherIdentity.AuthServer.Services.Csv;
+
+namespace TeacherIdentity.AuthServer.Pages.Admin;
+
+[Authorize(AuthorizationPolicies.GetAnIdentityAdmin)]
+public class AddUserImportModel : PageModel
+{
+    private readonly TeacherIdentityServerDbContext _dbContext;
+    private readonly IUserImportCsvStorageService _userImportCsvStorageService;
+
+    public AddUserImportModel(
+        TeacherIdentityServerDbContext dbContext,
+        IUserImportCsvStorageService userImportCsvStorageService)
+    {
+        _dbContext = dbContext;
+        _userImportCsvStorageService = userImportCsvStorageService;
+    }
+
+    [Required(ErrorMessage = "Select a file")]
+    [BindProperty]
+    public IFormFile? Upload { get; set; }
+
+    public void OnGet()
+    {
+    }
+
+    public async Task<IActionResult> OnPost()
+    {
+        if (!ModelState.IsValid)
+        {
+            return this.PageWithErrors();
+        }
+
+        if (Upload == null)
```

Comment:
Is this required as well as the `Required` attribute on the property?

Reply:
Removed